canonical-webteam/documentation-builder
# styling tweaks for small screen.

### Done
 - Menu button always stays in the top right at small and medium screens (title shrinks)
 - Breadcrumbs: swapped margin for padding to stop horizontal scrolling at small screen

### QA

`bin/documentation-builder --source-folder docs --search-url /search --force`
Check the generated pages at all viewports.

*Note: do not merge, needs code review*
